packages/@aws-cdk/aws-rds/test/integ.serverless-cluster-secret-rotation-custom-names.ts.snapshot/aws-cdk-rds-integ-secret-rotation.assets.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I have the same question for everything here and below: was this folder manually created and/or renamed? The snapshot should have only created a `js` folder, not a `ts` one. This PR has both.

Reply:
Hi @TheRealAmazonKendra. This folder was generated automatically after I created my integration test, and ran it by following the integration test guidelines mentioned [here](https://github.com/aws/aws-cdk/blob/main/INTEGRATION_TESTS.md).

Reply:
Can you clarify what the exact command you ran was? It should have only created a folder with the `js` type. I also ran it locally and only that folder was created.

Reply:
@TheRealAmazonKendra My bash history has been cleared, but recalling from memory, I ran `yarn integ --update-on-failed` on the new test file, followed by `yarn build`.

Reply:
Did you happen to run `integ --language typescript` at some point? Either way, the `ts` folder shouldn' be merged, only the `js` one. Can you delete both snapshot folders and rerun `yarn integ --update-on-failed`?